src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -82,9 +82,7 @@ public function sendResetLink(#[\SensitiveParameter] array $credentials, ?Closur
         // the current URI having nothing set in the session to indicate errors.
         $user->sendPasswordResetNotification($token);
 
-        if ($this->events) {
-            $this->events->dispatch(new PasswordResetLinkSent($user));
-        }
+        $this->events?->dispatch(new PasswordResetLinkSent($user));
```

Comment:
$this->events is not typed hinted as an \Illuminate\Contracts\Events\Dispatcher and because of this the proper check would be:
  
    if ($this->events instanceof \Illuminate\Contracts\Events\Dispatcher) {

But I don't expect this observation to be taken into consideration so take it as FYI only :)